drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -2806,6 +2806,7 @@ void RasterizerSceneGLES3::_copy_to_front_buffer(Environment *env) {
 		//no environment, simply convert from linear to srgb
 		storage->shaders.copy.set_conditional(CopyShaderGLES3::LINEAR_TO_SRGB,true);
 	} else {
+		/* Why are both statements equal? */
```

Comment:
Probably a bug. Would be worth checking the similar code in the gles2 rasterizer if it exists (some code was copy pasted and adapted, could have typos), or the git blame.